rollup.config.js

```diff
@@ -108,4 +109,4 @@ function generateConfig(configType) {
   return config;
 }
 
-export default [generateConfig('node'), generateConfig('browser')];
+export default [/*generateConfig('node'), */generateConfig('browser')];
```

Comment:
Why this change?

Reply:
Inadvertent, it helped me speed up my local testing. Removing